cluster/common.sh

```diff
@@ -347,7 +347,7 @@ function find-release-tars() {
 
   # This tarball is used by GCI, Ubuntu Trusty, and CoreOS.
   KUBE_MANIFESTS_TAR=
-  if [[ "${KUBE_OS_DISTRIBUTION:-}" == "trusty" || "${KUBE_OS_DISTRIBUTION:-}" == "gci" || "${KUBE_OS_DISTRIBUTION:-}" == "coreos" ]]; then
+  if [[ "${OS_DISTRIBUTION:-}" == "trusty" || "${OS_DISTRIBUTION:-}" == "gci" || "${OS_DISTRIBUTION:-}" == "coreos" ]]; then
```

Comment:
Because now that `OS_DISTRIBUTION` defaults to `gci`, one doesn't have to set `KUBE_OS_DISTRIBUTION` any more to start a GCI cluster. In that case, the old log would fail.

For other distros on this list, they all have to set `KUBE_OS_DISTRIBUTION` as before, but `OS_DISTRIBUTION` will always equal to `KUBE_OS_DISTRIBUTION`, so it should work for them too. Are you thinking of anything that could potentially break because of this change?